docs/modules/admin/pages/openapi.adoc

```diff
@@ -1,5 +1,9 @@
+////
+Module Type: REFERENCE
+////
+
 = OpenAPI
 
-The spog-api server includes an OpenAPI specification for the hosted policies, under the `/openapi.json` path.
+The `spog-api` server includes an OpenAPI specification for the hosted policies, under the `/openapi.json` path.
 
-Additionally, a UI for exploring the OpenAPI exists at `/swagger-ui/`.
+Additionally, a user interface for exploring the OpenAPI exists at `/swagger-ui/` directory.
```

Comment:
Where does the "swagger-ui" directory exist?  I don't see it in this repo.  Is this directory something that the user would have created locally?

Reply:
Same as with openapi, this is a HTTP url path on the trustification instance, so https://api.trustification.dev/swagger-ui/  for the public instance we're running.

Reply:
Thank you for clarifying.